tests/util/test_unit_conversion.py

```diff
@@ -559,110 +559,45 @@ def test_temperature_convert_with_interval(
 @pytest.mark.parametrize(
     "value,from_unit,expected,to_unit",
     [
-        (5, UnitOfVolume.LITERS, pytest.approx(1.32086), UnitOfVolume.GALLONS),
```

Comment:
So this is a bit of an unrelated change to this PR. Additionally, it should use approximation (it was recently added, actually).